frontend/e-buddy/src/app/(demo)/bots/view/page.tsx

```
@@ -0,0 +1,233 @@
+                  <ChevronUp className="h-4 w-4 text-gray-500" />
+                ) : (
+                  <ChevronDown className="h-4 w-4 text-gray-500" />
+                )}
+              </span>
+            </div>
+            <div className="col-span-3">Model</div>
+            <div
+              className="col-span-2 flex items-center cursor-pointer"
+              onClick={() => handleSort("createdAt")}
+            >
+              <span>Created</span>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)